### ISSUE TITLE
Is this an official release?

### ISSUE THREAD

**Issue by dandv** [rgrempel/isomorphic#1]
I don't see it mentioned at https://github.com/isomorphic-software or on http://github.smartclient.com/

**Comment by rgrempel** [comment]
No, it's not an official release. It was just a convenience for a few projects of mine. And, I haven't updated it in a very long time.
